crowdin/languages.go

```diff
@@ -0,0 +1,105 @@
+package crowdin
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/crowdin/crowdin-api-client-go/crowdin/model"
+)
+
+// LanguagesService handle communication with Languages methods.
+type LanguagesService struct {
+	client *Client
+}
+
+// List returns a list of all supported languages.
+// https://developer.crowdin.com/api/v2/#operation/api.languages.getMany
+func (s *LanguagesService) List(ctx context.Context, opts *model.ListOptions) ([]*model.Language, *Response, error) {
+	path := "languages?" + opts.Values().Encode()
+	req, err := s.client.NewRequest(ctx, http.MethodGet, path, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	body := new(model.LanguagesListResponse)
+	resp, err := s.client.Do(req, body)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	langs := make([]*model.Language, 0, len(body.Data))
+	for _, lang := range body.Data {
+		langs = append(langs, lang.Data)
+	}
+	return langs, resp, nil
```

Comment:
I agree that there is too much boilerplate code here, and it will be repeated in every method.
This approach follows the principles of the http client in the standard library (net/http). 

Need to consider options to reduce boilerplate code.

Reply:
Let's create some abstraction to reduce all this code, there are a lot of API endpoints and it's not okay to have all this duplicated in every method